### DISCUSSION TITLE
Issue on windows  -  Error: bash: command not found

### DISCUSSION THREAD

**Discussion by wyxcoder** [subosito/flutter-action#174] [General]
Run subosito/flutter-action@v2
Run $GITHUB_ACTION_PATH/setup.sh -c "C:\Users\...\actions-runner\_work\_tool/flutter" stable 2.[10](https://github.com/.../runs/6381901095?check_suite_focus=true#step:4:10).5 x64
  $GITHUB_ACTION_PATH/setup.sh -c "C:\Users\...\actions-runner\_work\_tool/flutter" stable 2.10.5 x64
  Error: bash: command not found

**Comment by ilovelinux** [comment]
Please post your workflow configuration

**Reply by saberkazyon** [reply]
@wyxcoder I'm facing the same issue right now did u manage to resolve it?

**Reply by prinkeeras** [reply]
same... is there any fix?

**Reply by super-brian** [reply]
on windows runner, installing Git (choco install git) would install git.exe in the same folder as git.exe. Make sure to do 'call RefreshEnv.cmd' after which do 'echo Path=%Path%>> %GITHUB_ENV%' so that git, bash, etc are available in the same step and others later.

**Reply by prabinlamsal19** [reply]
Hello, did you guys manage to resolve it? 
I am facing the same problem.
